services/src/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/FlowHistoryAction.groovy

```diff
@@ -0,0 +1,20 @@
+package org.openkilda.functionaltests.helpers
+
+enum FlowHistoryAction {
+    FLOW_CREATING("Flow creating"),
+    FLOW_UPDATING("Flow updating"),
+    FLOW_REROUTING("Flow rerouting"),
+    SET_THE_FLOW_STATUS_TO_DOWN("Set the flow status to down."),
+    FLOW_WAS_VALIDATED_SUCCESSFULLY("Flow was validated successfully")
```

Comment:
This is fine, but you are not supposed to always duplicate the phrase word-to-word. `FLOW_VALIDATION_SUCCESS` would work as well